### DISCUSSION TITLE
Can geckos.io be used for a full duplex connection (data and video) to/from an unmanned vehicle?

### DISCUSSION THREAD

**Discussion by pratikc192** [geckosio/geckos.io#211] [Help / Questions]
I want to make a 4G connected FPV drone by setting up a server on the vehicle itself using a rasp pi, and connecting (p2p) it to a web app for controls?

**Comment by yandeu** [comment]
Only data; no video.

**Reply by pratikc192** [reply]
Is there anyway I can encode the video and send using the data channel?
